homeassistant/components/apns/notify.py

```diff
@@ -19,12 +18,12 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.event import track_state_change
 
+DOMAIN = "apns"
```

Comment:
We should import the DOMAIN from local `const` or `init` and not defined it on platform